configure.ac

```diff
@@ -352,6 +352,33 @@ else
     AC_MSG_RESULT([yes])
 fi
 
+##########################
+# KVM vmapoll workaround
```

Comment:
It is obvious that runtime method is flexible but it might decrease performance for both scenarios as KVM and non KVM using additional check on fast flow.
I think that final solution should be based on points as:
- if KVM usage is primary way;
- if there is possibility to recommend end user to build vma from source basing release tag;

May be it is enough to use direct copy macro (w/o asm) for both scenarios but in this case we need results of performance measurements comparing two versions of copy macro.